rocks.noise.QSpeakers.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: rocks.noise.QSpeakers
+runtime: org.kde.Platform
+runtime-version: '6.7'
```

Comment:
current is 6.8

Reply:
thanks, I am testing a local rebuild with kde platform 6.8